sonyapilib/device.py

```diff
@@ -659,6 +659,21 @@ def get_apps(self):
         """Get the apps from the stored dict."""
         return list(self.apps.keys())
 
+    def volume_up(self):
+        # pylint: disable=invalid-name
+        """Sends the command 'VolumeUp' to the connected device."""
+        self._send_command('VolumeUp')
+
+    def volume_down(self):
+        # pylint: disable=invalid-name
+        """Sends the command 'VolumeDown' to the connected device."""
+        self._send_command('VolumeDown')
+
+    def mute_volume(self):
```

Comment:
Can the command be named "mute" this would eliminate the need for the case in the tests

Reply:
yes but i need to update media player althougnt..

Reply:
I think you have to open a new PR in the media player repository to implement the changed name because I closed it already :( sorry

Reply:
Done ;)